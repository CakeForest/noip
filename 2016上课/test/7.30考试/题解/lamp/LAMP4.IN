100
0 50 42 9719
1 3 9 9148
1 10 37 7698
1 15 7 8182
2 30 6 5299
3 23 26 7007
5 32 2 1856
3 27 15 8628
5 2 3 2515
5 41 4 1558
5 4 4 4695
10 27 1 1875
9 48 1 7123
6 30 4 2030
6 1 1 2102
8 45 11 3525
8 43 7 5473
12 31 1 9411
10 32 3 2685
2 25 6 2519
9 25 1 9260
9 38 2 4569
12 30 1 286
15 30 1 3063
17 36 6 1999
9 49 1 5023
4 26 3 9702
13 17 1 2611
15 9 1 5830
28 31 1 3327
5 21 4 461
1 18 4 134
23 44 1 9735
26 4 1 2526
19 38 1 6004
15 49 1 1390
23 21 1 2761
3 40 18 6671
15 23 1 8108
7 23 1 413
20 10 1 8356
14 25 2 6839
24 39 1 8895
34 38 1 8948
19 16 1 8392
29 37 1 5655
21 38 1 5278
41 23 1 8036
16 16 3 7987
40 43 1 5753
23 38 1 3175
1 6 23 8482
37 41 1 4707
19 10 2 174
6 31 20 1350
19 43 1 9416
39 41 1 8246
48 29 1 5149
51 37 1 7212
26 15 1 6666
27 47 2 9186
13 6 1 8244
6 43 12 9421
18 41 1 5172
56 20 1 7911
41 23 1 7337
55 40 8 4646
17 4 4 2142
30 44 1 7290
29 9 1 8397
6 44 14 8316
23 43 1 188
19 6 2 5213
7 25 1 2521
66 13 1 1709
3 17 3 2144
71 16 4 5566
16 26 1 5217
66 0 1 224
1 23 30 1798
57 25 1 8758
80 23 5 3387
70 47 1 175
66 43 1 7246
2 25 6 5374
59 24 1 7986
75 25 1 3961
46 43 1 6095
42 41 2 9063
12 31 1 2821
61 9 2 5937
61 0 1 4230
71 23 10 7243
85 12 2 2465
9 46 2 3856
73 34 1 2362
14 11 4 729
52 25 2 4746
71 11 5 3201
6 15 16 7082
