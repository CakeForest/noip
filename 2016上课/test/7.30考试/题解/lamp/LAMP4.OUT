5 43229
1 2 3 4 52
